### ISSUE TITLE
Install NCPA on ubuntu 18.04 problem

### ISSUE THREAD

**Issue by Kesslarr** [NagiosEnterprises/ncpa#700]
Hi there,
I am new to ncpa, I am installing ncpa with [guide](https://github.com/NagiosEnterprises/ncpa/blob/master/BUILDING.rst#building-on-linux)

I do this command as following
`git clone https://github.com/NagiosEnterprises/ncpa`

next step is
```
cd ncpa/build/scripts
./linux_build_prereqs.sh
./linux_build_setup.sh
```

but my question is that I didn't find `scripts` folder in `build`....
![image](https://user-images.githubusercontent.com/44806366/95308829-f3ace400-08bc-11eb-91bb-148c7400d7d8.png)

I go to linux folder, and it has these files
![image](https://user-images.githubusercontent.com/44806366/95309083-44bcd800-08bd-11eb-872d-e33a885f521d.png)

I'm not sure which script I have to run....

Can anyone help me? Thanks.

**Comment by ccztux** [comment]
You are trying to build ncpa. If you want to install it, follow [the getting started guide](https://www.nagios.org/ncpa/getting-started.php#linux).

**Comment by Kesslarr** [comment]
> You are trying to build ncpa. If you want to install it, follow [the getting started guide](https://www.nagios.org/ncpa/getting-started.php#linux).

Hi @ccztux ,
thanks for your answering, I followed this guide, and it works well.

I want to ask another question, I'm using nagios before ncpa, and I also a newbie with nagios.
I want to monitor remote devices, such as cpu/gpu usage, memory, storage..., I took one as nagios server(A), and others are monitored by server.
But I found out that if I want to use nagios to monitor the infomation I want, I have to install extra plugins, and plugins I found were old or not compatible with my nagios version(nagios core version4.4.6)
I want to ask that can I use nagios to manage ncpa? and I want to use ncpa to see all my remote devices information, is that mean I have to install ncpa in all remote devices or I can install in one device as server and use this to monitor others?
